service-discovery/consul/src/main/java/io/smallrye/stork/servicediscovery/consul/ConsulServiceDiscovery.java

```diff
@@ -1,7 +1,11 @@
 package io.smallrye.stork.servicediscovery.consul;
 
+import static io.smallrye.stork.servicediscovery.consul.ConsulMetadataKey.*;
```

Comment:
someone told me that this is bad as the _death star_ :-D

Reply:
Gr gr gr, it slipped through the cracks